modules/guides/pages/javascript-udfs/data-handling-javascript-udf.adoc

```diff
@@ -0,0 +1,149 @@
+= Data handling in Javascript UDFs
```

Comment:
Haven't gone into this page in detail but the same comments apply. This is a tutorial rather than a how-to guide.

Reply:
Removed page